gtdynamics/cablerobot/src/test_cdpr_planar_sim.py

```diff
@@ -51,7 +51,7 @@ def update(self, values, k):
         xdot = 0
         for k in range(10):
             pExp = Pose3(Rot3(), (x, 0, 1.5))
-            self.gtsamAssertEquals(pExp, pAct[k], tol=0)
+            self.gtsamAssertEquals(pExp, pAct[k], tol=1e-15)
```

Comment:
Ya, I guess my personal tendency is to explicitly define the tolerances to give more semantic information to the reader (e.g. this tolerance is really small so it's some sort of "equality" rather than an unconstrained optimization or something).
In the context of this code, semantically simulation should have the smallest tolerance, dynamics during LQR should have very small tolerance, and Q/R can have relatively large tolerance compared to analytical solution.

But I do recognize that this can make the tests more brittle, so I'll keep this in mind in the future.